cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -383,12 +383,12 @@ private synchronized void removeOldUserTasks() {
   }
 
   /**
-   * Mark the user task which is currently executed in {@link com.linkedin.kafka.cruisecontrol.executor.Executor}.
+   * Update status for user task which is currently executed in {@link com.linkedin.kafka.cruisecontrol.executor.Executor}.
    *
    * @param uuid UUID associated with the in-execution user task.
    * @return {@link UserTaskInfo} associated with the task in-execution.
    */
-  public synchronized UserTaskInfo markTaskInExecution(String uuid) {
+  public synchronized UserTaskInfo taskExecutionBegan(String uuid) {
```

Comment:
Nit: The name change `markTaskInExecution` -> `taskExecutionBegan` makes the functionality of the method not clear (e.g. is it a getter or setter)?